openghg/store/_obssurface.py

```diff
@@ -683,3 +686,33 @@ def seen_hash(self, file_hash: str) -> bool:
 
     def set_hash(self, file_hash: str, filename: str) -> None:
         self._file_hashes[file_hash] = filename
+
+    @staticmethod
```

Comment:
Can I ask if there a reason why this needs to be a static method do you think? For `validate_data` and `scheme` these were static methods so they could be called on the class without needing to create a specific instance but I don't think that would the case for this method?

Reply:
It functions as a utility for the class so created it as a staticmethod.

Reply:
I guess `self` would be an unused argument in this function, so a staticmethod makes sense. Inside `ObsSurface`, you can use `self.my_static_method` instead of `ObsSurface.my_static_method`.

Reply:
I don't think this particular function as written would be intended for use outside of a specific instance of the class in this way e.g. we wouldn't expect to call `ObsSurface.align_metadata_attributes(data)` (especially at the moment given that `data` is a more complicated object). We may later on want to pull some of the details out of this but I don't think we'd use the method in this form as a utility function.

I don't think that not using `self` at the moment within the function necessarily means this would be better as a static method?